src/main/java/com/softserve/dto/ArticleDTO.java

```diff
@@ -0,0 +1,19 @@
+package com.softserve.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class ArticleDTO {
+
+	private Integer idArticle;
```

Comment:
any reason why not using raw types? (int instead of Integer)

Reply:
I usually use wrapper for entity style